samples/wifi/shell/src/wifi_main.c

```diff
@@ -1,11 +1,19 @@
 /*
- * Copyright (c) 2012-2014 Wind River Systems, Inc.
+ * Copyright (c) 2022 Nordic Semiconductor ASA
  *
- * SPDX-License-Identifier: Apache-2.0
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/** @file
+ * @brief WiFi shell sample main function
  */
 
 #include <zephyr/sys/printk.h>
 #include <nrfx_clock.h>
+#ifdef CONFIG_NET_L2_OPENTHREAD
```

Comment:
I guess we can remove this conditional flag...i.e. make this applicable by default.